### PR TITLE
1053755 api test automation and doc

### PR DESCRIPTION
**Pull request by vijijustin**
Review changes... Contacts and Users endpoints

### CODE REVIEW

File: api/planmill1_5.raml [hunk 1]
```diff
@@ -139,6 +140,19 @@ traits:
                   "message": "Invalid date pattern",
                   "code": "00500"
                 }
+  - some:
```
Comment (vijijustin) [inline api/planmill1_5.raml]: I don't see why its being modified or added. developer link has the same content right now.

File: api/user1_5.sample [hunk 2]
```diff
@@ -1,9 +1,5 @@
 Note:
```
[the remark this answers is not in the document]
Comment (Tinbit) [reply api/user1_5.sample]: Already gave feedback reply on the request.But, again please check RAML basic Structure first.

[the remark this answers is not in the document]
Comment (Tinbit) [reply api/user1_5.sample]: Already gave feedback reply on the request.But, again please check RAML basic Structure first.

File: api/planmill1_5.raml [hunk 3]
```diff
@@ -443,7 +443,7 @@ resourceTypes:
         responses:
           204:
             description: No HTTP body returned
-  - putDeleteContactCollectionItem:
+  - putContactCollectionItem:
```
[the remark this answers is not in the document]
Comment (Tinbit) [reply api/planmill1_5.raml]: OK, I will rename it to putCollectionItem.

File: api/timecard1_5.sample [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
-    "date": "2016-10-15T00:00:00.000+0300",
-    "totalFinishTime": 1080,
-    "totalStartTime": 600,
-    "person": 3505,
-    "breakAmount": 90,
-    "workStartTime": 600,
-    "workFinishTime": 1080
+    "date": "2019-02-28T00:00:00.000+0200",
```
[the remark this answers is not in the document]
Comment (Tinbit) [reply api/timecard1_5.sample]: Remember samples are not done manually. All samples are response results and noone can go to the specific date field and change it back manually to the old sample date. Please give feedback on what is necessary.

File: api/planmill1_5.raml [hunk 5]
```diff
@@ -459,6 +459,12 @@ resourceTypes:
         responses:
           204:
             description: No HTTP body returned
+  - deleteCollectionItem:
```
[the remark this answers is not in the document]
Comment (Tinbit) [reply api/planmill1_5.raml]: Yeah It's unused collection item, it should be removed.Missed it when code cleaning.

File: api/planmill1_5.raml [hunk 6]
```diff
@@ -717,9 +727,11 @@ resourceTypes:
           schemaName:
           exampleCollectionItemArray: !include tempfile_array1_5.sample
       /{attachment_id}:
+        delete:
+          description: Remove a single attachment from PlanMill.
         type:
-          putGetDeleteCollectionItem:
-            exampleCollectionItem: Contents of file
+          deleteOnlyCollectionItem:
```
[the remark this answers is not in the document]
Comment (Tinbit) [reply api/planmill1_5.raml]: Yeah It's unused collection item, it should be removed.Missed it when code cleaning.

File: api/planmill1_5.raml (hunk 1 above)
Comment (Tinbit) [reply api/planmill1_5.raml]: It is NOT the same at all. Compare understand it well. I will ignore this review comment.

File: api/contact1_5.sample [hunk 7]
```diff
@@ -1,36 +1,37 @@
...
-  "mobilePhone": "+358-466-5556951",
-  "workPhone": "+358 10 322 9110",
-  "googleTalk": "gtalk_id",
-  "primaryAddress": "Hämeentie 19",
-  "account": 51
+    "lastName": "Tester",
+    "secondaryAddress": null,
+    "primaryState": "",
+    "secondaryEmail": "",
+    "role": null,
```
[the remark this answers is not in the document]
Comment (Tinbit) [reply api/contact1_5.sample]: It's a response result. It's not manually created.Ignored.

**Comment by vijijustin** [comment]
Reviewed

**Pull request merged**